pkg/apis/security/v1alpha1/types.go

```diff
@@ -35,6 +35,10 @@ type NetworkPolicy struct {
 
 // NetworkPolicySpec defines the desired state for NetworkPolicy.
 type NetworkPolicySpec struct {
+	// Tier specifies the tier to which this NetworkPolicy belongs to.
+	// The NetworkPolicy order will be determined based on the combination of the
+	// Tier's Priority and the NetworkPolicy's own Priority.
+	Tier string `json:"stirng"`
```

Comment:
Is it optional or mandatory? I thought it can be optional, and if not set the default (lowest) tier will be used. This is to be compatible with the current version, and also make it more convenient for single tier cases.

Reply:
makes sense.. i started with mandatory.. but we could by default assign it to lowest priority tier (still above K8s NP tier) when there is no tier mentioned..

Reply:
updated